### DISCUSSION TITLE
The provided Place ID is no longer valid

### DISCUSSION THREAD

**Discussion by Emiliano-Bucci** [wellyshen/use-places-autocomplete#1140] [Q&A]
@wellyshen  Hi! I'm getting the following error: 
The provided Place ID is no longer valid. Please refresh cached Place IDs as per https://developers.google.com/places/web-service/place-id#save-id

Looking into the docs it says that we can refresh the result by making a specific call; is this supported in the library?

Thanks!

**Comment by luddilo** [comment]
Getting the same thing now all of a sudden. Any input please @wellyshen ?
